chrome/content/zotero/integration/quickFormat.js

```diff
@@ -70,8 +70,13 @@ var Zotero_QuickFormat = new function () {
 			// Only hide chrome on Windows or Mac
 			if(Zotero.isMac) {
 				document.documentElement.setAttribute("drawintitlebar", true);
-			} else if(Zotero.isWin) {
-				document.documentElement.setAttribute("hidechrome", true);
+			}
+	
+			// Required for dragging to work on windows.
+			// These values are important and changing them may affect how the dialog
+			// is rendered
+			if (Zotero.isWin) {
+				document.documentElement.setAttribute('chromemargin', '0,0,15,0');
 			}
```

Comment:
How do you reproduce whatever issue this is solving?

Reply:
If I do `document.documentElement.setAttribute('chromemargin', '0,2,2,2');` (same as the main zotero pane window) then I see the dialog disappearing like [this](https://www.dropbox.com/scl/fi/9dyp9n52ml03uohz7nu36/panel_gone.mov?rlkey=wcp8csgprfg8oety9ec4yg1uh&dl=0) on windows 11 VM. 

My observation is that changing the first digit (top margin) to anything but `0` as well as decreasing the bottom margin to lower than `15` (I tried 10, 5 and 2) causes such the breakage.

Reply:
So this kind of sucks. There's too much hackery interfering with itself in this dialog. But let's just make this work for fx115 and I will look to make it less complicated for the redesign.